docs/command_line_tools.md

```diff
@@ -279,6 +280,44 @@ Transaction hash: 0xb00afc5c2bb92a76d03e17bd3a0175b80609e877cb124c02d19000d52939
 Mined block number: 1849039
 ```
 
+Auditing Tools
+------------
+
+Smart contracts written in Solidity often include logic bugs and other issues which might compromise their security. These are not always obvious to programmers. Issues can include [integer precision problems](https://github.com/smartdec/classification#arithmetic), [re-entrancy attacks](https://github.com/smartdec/classification#contract-interaction), and many other flaws. As Ethereum smart contracts are immutable once they have been deployed, it is important that they are bug-free at this point. Web3j is able to audit smart contracts for certain common issues and vulnerabilities using static code analysis. 
+
+To audit a smart contract:
+
+``` bash
+$ web3j audit <path to contract file>
+```
+
+An example of the output from this command is as follows:
+``` bash
+              _      _____ _     _        
```

Comment:
Sorry one other thought - we should include `$ web3j audit Campaign.sol` to the example is more explicit